Classes/MMNumberKeyboard.m

```diff
@@ -424,6 +424,35 @@ - (void)setReturnKeyButtonStyle:(MMNumberKeyboardButtonStyle)style
     }
 }
 
+- (void)setReturnKeyFont:(UIFont *)returnKeyFont
+{
+    if ([_returnKeyFont isEqual:returnKeyFont]) {
+        return;
+    }
+
+    _returnKeyFont = returnKeyFont;
+
+    _MMNumberKeyboardButton *button = self.buttonDictionary[@(MMNumberKeyboardButtonDone)];
+    [button.titleLabel setFont:returnKeyFont];
+}
+
+- (void)setKeypadFont:(UIFont *)keypadFont
+{
+    if ([_keypadFont isEqual:keypadFont]) {
+        return;
+    }
+
+    _keypadFont = keypadFont;
+
+    const NSInteger numberMin = MMNumberKeyboardButtonNumberMin;
+    const NSInteger numberMax = MMNumberKeyboardButtonNumberMax;
```

Comment:
Probably not. This was to mimic the way similar code was written to increase the chance of merging it upstream, before I learned he's not interested in this kind of feature.